instances/experimentalApproach/culturing.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/experimentalApproach/culturing",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/ExperimentalApproach",
+  "definition": "'Culturing' covers any experimental approach that focuses on breeding, care, and keeping of living things in order to harvest the substances they produce.",
```

Comment:
That definition is closer to husbandry. I don't have a good definition off the top of my head here, will think on it.

Reply:
thanks. I really wasn't sure what you all want to include here.